_includes/maintenance-banner-service-disruption.html

```diff
@@ -0,0 +1,15 @@
+<section aria-label="Site alert" class="usa-site-alert usa-site-alert--emergency">
+  <div class="grid-container padding-0">
+    <div class="usa-alert">
+      <div class="usa-alert__body">
+        <h3 class="usa-alert__heading text-white">
+          Service disruption
+        </h3>
+        <p class="usa-alert__text">
+          date/time: We're investigating service disruption on the .gov registrar. The .gov zone and individual domains remain online. However, the registrar is running slower than usual.
```

Comment:
```suggestion
          date/time: We're investigating a service disruption on the .gov registrar. The .gov zone and individual domains remain online. However, the registrar is running slower than usual.
```

Reply:
what date/time format do we want people to use? I want to avoid any confusion here if someone is dealing with a critical situation. I think an example is ideal here.

Reply:
Jan 30, 17:27 UTC

Reply:
OK, using my birthdate as a placeholder